contracts/assetListing/AssetListingGUni.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.7.5;
+pragma abicoder v2;
+
+import {IAaveOracle} from "./interfaces/IAaveOracle.sol";
+
+interface IPoolConfigurator {
```

Comment:
Could you rename it so its clear is the one of the AMM Market?

Reply:
👍